package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "cli": "ts-node --require ./app/test/globals.ts --require ./app/src/cli/dev-commands-global.js app/src/cli/main.ts",
     "test:integration": "cross-env TEST_ENV=1 ELECTRON_NO_ATTACH_CONSOLE=1 xvfb-maybe mocha -t 30000 --require ts-node/register app/test/integration/*.ts",
-    "test:unit": "cross-env GIT_AUTHOR_NAME=\"Joe Bloggs\" GIT_AUTHOR_EMAIL=\"joe.bloggs@somewhere.com\" GIT_COMMITTER_NAME=\"Joe Bloggs\" GIT_COMMITTER_EMAIL=\"joe.bloggs@somewhere.com\" TEST_ENV=1 ELECTRON_NO_ATTACH_CONSOLE=1 xvfb-maybe electron-mocha -t 10000 --renderer --require ts-node/register --require ./app/test/globals.ts app/test/unit/*.{ts,tsx} app/test/unit/**/*.{ts,tsx}",
+    "test:unit": "cross-env GIT_AUTHOR_NAME=\"Joe Bloggs\" GIT_AUTHOR_EMAIL=\"joe.bloggs@somewhere.com\" GIT_COMMITTER_NAME=\"Joe Bloggs\" GIT_COMMITTER_EMAIL=\"joe.bloggs@somewhere.com\" TEST_ENV=1 ELECTRON_NO_ATTACH_CONSOLE=1 DEBUG=xvfb-maybe ./node_modules/.bin/xvfb-maybe electron-mocha -t 10000 --renderer --require ts-node/register --require ./app/test/globals.ts app/test/unit/*.{ts,tsx} app/test/unit/**/*.{ts,tsx}",
```

Comment:
![](https://user-images.githubusercontent.com/359239/37454755-7dbf1a34-288f-11e8-9dfe-15decdbbba09.gif)

So, something that's changed between yarn `1.3.2` and `1.5.1` is that this trick no longer works:

```shellsession
... xvfb-maybe electron-mocha ...
```

`xvfb-maybe` is designed to just spawn the arguments it receives, so it's unable to resolve `electron-mocha` (`npm` should be adding `node_modules/.bin` to the `PATH`, but clearly it's not) and so adding the path segment to either command here is the workaround:

```shellsession
... ./node_modules/.bin/xvfb-maybe electron-mocha ...
```


I'll dig into this more tomorrow.

Reply:
`yarn xvfb-maybe` might work too.